front/public/locales/fr/operationalStudies/manageTrainSchedule.json

```
@@ -53,6 +53,21 @@
   "infraLoading": "Infra en cours de chargement…",
   "inputOPTrigrams": "Entrez les trigrammes des points remarquables par lesquels vous voulez que l'itinéraire se fasse, séparés par un espace.",
   "inputOPTrigramsExample": "Ex : LSN CIG CCS",
+  "incompatibleConstraints": {
+    "filters": {
+      "title": "Filtrer sur le type de contrainte",
+      "enable": "Afficher les contraintes de type '{{name}}'",
+      "disable": "Cacher les contraintes de type '{{name}}'"
+    },
+    "incompatible_electrification_ranges": "Électrification incompatible",
+    "incompatible_gauge_ranges": "Gabarit incompatible",
+    "incompatible_signaling_system_ranges": "Système de signalisation incompatible",
+    "info": "La recherche de chemin a échoué à cause de certaines contraintes. Vous trouverez ici la liste des contraintes ayant échoué sur le chemin le plus court, avec pour chacune, le type de contrainte, la position du début de la zone d’échec par rapport au début du chemin, et sa longueur. ​",
+    "gotoMap": "Voir sur la carte",
+    "seeConstraintsOnMap": "Voir la liste des contraintes incompatibles sur la carte",
```

It's not really the list that we will see when clicking here.

```suggestion
    "seeConstraintsOnMap": "Voir toutes les contraintes incompatibles sur la carte",
```

Other suggestion (you can adapt english trad and the `center on itinerary` if you chose this one)

```suggestion
    "seeConstraintsOnMap": "Centrer sur toutes les contraintes incompatibles",
```

---

You will see the filtered list on the map, not everything.  So for me it can't be `toutes les contraintes`. 
We can just say `Voir les contraintes`.